tv.html

```diff
@@ -0,0 +1,239 @@
+---
+layout: default
+---
+
+<section class="bg-tan-light soft-top">
+  <div class="container soft-top">
+    <div class="col-md-8">
+      <div class="latest-message">
+        <div class="latest-message-background soft-half-bottom push-bottom">
+          <div class="embed-container">
+            <div class="media-video">
+              <div class="card-video-player">
+                <div class="embed-responsive embed-responsive-16by9" id="js-media-video" player-id="player"
+                  video-id="9vmwUS1E-Lc" data-video-player>
+                  <div id="player"></div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <h2 class="latest-message-headline font-family-condensed-extra text-uppercase flush-top text-gray-dark">
+            Is God Good? | The Good God Journey Week 1
+          </h2>
+        </div>
+      </div>
+    </div>
+    <div class="col-sm-4">
+      <div class=" bg-charcoal media featured soft-sides">
+        <div class="media-body soft-bottom">
+          <h2 class="text-uppercase font-family-serif font-size-base font-size-small text-white">current series</h2>
+          <a href="https://www.crossroads.net/media/series/the-good-god-journey" class="push-half-bottom"
+            data-automation-id="series-image">
+
+            <img class="full-width"
+              src="https://crds-media.imgix.net/5a5Jw5jhgnTeSjdyHejpWO/791786a39157f100723274cadc0bc73a/journey2020-good-god.jpg?{{ site.imgix_params.placeholder_sixteen_nine }}"
+              sizes="{{ site.image_sizes.one_fourth }}" data-optimize-img />
+          </a>
+          <h5 class="component-header font-size-h3 text-white">The Good God! Journey</h5>
+          <div class="soft-quarter-ends font-size-smaller push-bottom text-white">
+            2020 has been hard on everyone. A journey is a chance to reconnect, learn, and grow together. The Journey
+            might just be the best thing you do all year.
+          </div>
+          <crds-button href="https://www.crossroads.net/media/series/the-good-god-journey" color="white"
```

Comment:
Why can't this be `href="/media/series/the-good-god-journey"`? Is it because it only lives in prod? There are also a lot of other hrefs that I have the same question about